tests/django/manage.py

```diff
@@ -29,4 +29,7 @@
 
     from django.core.management import execute_from_command_line
 
+    os.environ.setdefault(u'HYPOTHESIS_DJANGO_USETZ_TEST', u'TRUE')
```

Comment:
This looks like a trap waiting to happen and confuse us to me. I'd rather do this from the script that calls it than in the manage.py.

Particularly because I'm not entirely sure I believe that the settings get reloaded if you call this twice.